SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -155,39 +155,27 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 		return NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::GetClientAuthorityComponent(GetDefault<USpatialGDKSettings>()->UseRPCRingBuffer()));
 	}
 
-	// Indicates whether this client worker has "ownership" (authority over Client endpoint) over the entity corresponding to this channel.
-	FORCEINLINE bool IsAuthoritativeClient() const
+	inline void OnClientAuthorityChange(Worker_AuthorityChangeOp Op)
```

Comment:
```suggestion
	inline void OnClientAuthorityChange(const Worker_AuthorityChangeOp& Op)
```

Reply:
maybe you don't need to make it a ref if inlining magic figures that out but idk

Reply:
done